Gruntfile.js

```diff
@@ -73,9 +75,39 @@ module.exports = function (grunt) {
     grunt.loadNpmTasks('grunt-curl');
     grunt.loadNpmTasks('grunt-zip');
     grunt.loadNpmTasks('grunt-file-creator');
-    grunt.loadNpmTasks('grunt-npm-install');
     grunt.loadNpmTasks('grunt-webpack');
 
+    grunt.registerTask('npm-install', 'Install plugin NPM dependencies', function (plugin) {
```

Comment:
Since this task is to do with plugins, we should move it inside plugin.js IMO, to keep this top level Gruntfile clean.